lesion_segmentation_protocol.md

```diff
@@ -0,0 +1,31 @@
+# Lesion segmentation protocol:
+
+The following details the protocol for Multiple Sclerosis (MS) lesion segmentation in the spinal cord.
+Imaging the spinal cord is often essential to confirm the diagnosis of MS. That is because the lesions of the spinal cord are included in the McDonald diagnosis criteria, which studies dissemination in space and in time [(Thompson et al. 2018)](https://pubmed.ncbi.nlm.nih.gov/29275977/). While the MAGNIMS-CMSC-NAIMS working group recommends to use at least two sagittal images for MS diagnosis, still, axial imaging is mentioned as optional in international imaging guidelines [(Wattjes et al. 2021)](https://pubmed.ncbi.nlm.nih.gov/34139157/).
+For detecting MS lesions in the spinal cord, two main contrasts emerge: PSIR and STIR contrasts. New studies [(Peters et al. 2024)](https://pubmed.ncbi.nlm.nih.gov/38289376/)[(Fechner et al. 2019)](https://pubmed.ncbi.nlm.nih.gov/30679225/) showed that using PSIR contrasts improved MS lesion detection in the spinal cord. [(Fechner et al. 2019)](https://pubmed.ncbi.nlm.nih.gov/30679225/) showed that the PSIR contrast showed a higher signal-to-noise (SNR) ratio compared to the STIR contrast. 
+
+## Criteria to segment MS lesions in the spinal cord:
```

Comment:
```suggestion
## Criteria to segment MS lesions in the spinal cord:
```
I would add a clear definition of what consists a lesion according to the sequences you are using (in effort for standardization)

Reply:
This is a great idea ! Could you propose a definition ?

Reply:
```suggestion
## Criteria to segment MS lesions in the spinal cord:
Based on the definition proposed for MS lesions in the spinal cord in the 2017 revisions of the
McDonald criteria:
"A hyperintense lesion in the cervical, thoracic, or lumbar spinal cord seen on T2 plus short tau inversion recovery, proton-density images, or other appropriate sequences, or in two planes on T2 images."  [(Thompson et al. 2018)](https://pubmed.ncbi.nlm.nih.gov/29275977/) [(Filippi et al. 2016)](https://pubmed.ncbi.nlm.nih.gov/26822746/) [(Brownlee et al. 2018)](https://pubmed.ncbi.nlm.nih.gov/27889190/) [(Rovira et al. 2016)](https://pubmed.ncbi.nlm.nih.gov/26149978/) 
```

Reply:
Definition added. !